homeassistant/components/govee_light_api/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "govee_light_api",
+  "name": "Govee Lights - Local API",
```

```suggestion
  "domain": "govee_light_local",
  "name": "Govee lights local",
```
Discussed with some people and we came up with these names. If we can change this in this PR, that'd be great